src/Resources/views/CRUD/edit_orm_one_to_many_inline_table.html.twig

```diff
@@ -22,7 +22,8 @@ file that was distributed with this source code.
                         {% if nested_field.vars['required'] %}
                             class="required"
                         {% endif %}
-                        {% if (nested_field.vars['attr']['hidden'] is defined) and (nested_field.vars['attr']['hidden']) %}
+                        {% if (((nested_field.vars['attr']['hidden'] is defined) and (nested_field.vars['attr']['hidden']))
```

Comment:
The extra parens you are adding are unnecessary because of operator precedence I think

Reply:
Yes, first is totally unnecessary (habit from PHP). But second makes it more friendly to read.